mapping/requirements_officejs.json

```diff
@@ -7210,6 +7210,17 @@
 								}
 							}],
 							"availability": "GA"
+						},
+												{
+							"name": "Mailbox",
+							"apiVersion": "1.14",
```

Comment:
double checking - the commit comment says it's for 1.13?

Reply:
my bad, it is for 1.14